index.ios.js

```diff
@@ -1,13 +1 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- */
-
-import App from './lib/app';
-import { AppRegistry } from 'react-native';
-
-class Root extends App {
-
-}
-
-AppRegistry.registerComponent('Magnet', () => Root);
+require('./index.android.js');
```

Comment:
Then i prefer to leave it as it is, the meaning is clear, same entry point for ios and android :P